ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -3601,6 +3601,24 @@ AHU_DFSS_DXZC_ECONZ_EFSS2X_HT2ZC_CO2C:
   - HT2ZC
   - CO2C
 
+AHU_CHWSC_EFVSC_FDPM3X_HTWHLSTC_HWSC_RCO2M_RSPC_SFM_SFVSC_SSPC_VOADM_VMADC:
+  description: "Complex multi zone AHU with heat recovery wheel and recirculated air control."
+  is_canonical: true
+  implements:
+  - DOAS
+  - CHWSC
+  - EFVSC
+  - FDPM3X
+  - HTWHLSTC
+  - HWSC
+  - RCO2M
+  - RSPC
+  - SFM
+  - SFVSC
```

Comment:
it may be redundant, but precendent states whenever `SFVSC` or `EFVSC` are implemented that the basic SS types be included, as well (`SFSS`, `EFSS`)